Source/MediaInfo/File__Analyze.cpp

```diff
@@ -2218,12 +2218,15 @@ bool File__Analyze::Header_Manage()
         }
         else
         {
+            if(Element_Level-2 >= Element.size())

```

Comment:
but... if Element.size() == 0?

Reply:
Element has a fixed size and can not be 0 (nothing would work if it is the case).
And here your test works only because Element_Level-2 is less than 0 so 2^64-2 (unsigned integer), if I change it to signed integer it would not work anymore.